compiler/rustc_session/src/config.rs

```diff
@@ -1441,14 +1439,7 @@ impl<'a, T: Eq + Hash + Copy + 'a> Extend<&'a T> for ExpectedValues<T> {
     }
 }
 
-/// Converts the crate `--check-cfg` options from `String` to `Symbol`.
-/// `rustc_interface::interface::Config` accepts this in the compiler configuration,
-/// but the symbol interner is not yet set up then, so we must convert it later.
```

Comment:
Do you think it is worth keeping this comment? by moving it to the `intern` function maybe.

Reply:
There are other new comments that communicate the same ideas, so I think that's good enough.